Canillitapp Widget/TodayViewController.swift

```diff
@@ -33,36 +33,33 @@ class TodayViewController: UIViewController, NCWidgetProviding {
 
         showActivity(true)
 
-        let success: (TopicList?) -> Void = { [weak self] (topicList) in
-            guard let self = self else {
-                return
-            }
+        let handler: ((Result <TopicList?, Error>) -> Void) = { [weak self] result in
+            switch result {
+            case .success(let topicList):
+                guard let self = self else {
+                    return
+                }
 
-            self.showActivity(false)
+                self.showActivity(false)
 
-            self.todayViewModel.topics = topicList?.topics
+                self.todayViewModel.topics = topicList?.topics
 
-            guard let attributedText = self.todayViewModel.attributedTitlesString else {
-                return
-            }
+                guard let attributedText = self.todayViewModel.attributedTitlesString else {
+                    return
+                }
 
-            self.widgetLabel.attributedText = attributedText
-        }
+                self.widgetLabel.attributedText = attributedText
 
-        let fail: (NSError) -> Void = { [weak self] (error) in
-            guard let self = self else {
-                return
-            }
+            case .failure(_):
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/cafeb7b8636cbafd0216eaa2489bb108aa52959c/Canillitapp Widget/TodayViewController.swift#L53"></span>Arguments can be omitted when matching enums with associated types if they are not used.<br />
<code>empty_enum_arguments</code> <code>TodayViewController.swift:53</code></td>
    </tr>
  </tbody>
</table>